### PR TITLE
Fix sorted lambda for old STRs

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

**How to test**:
Mark an old STR causative e.g. on stage: note crash with master, and working with patch:
Screenshot 2021-05-17 at 15 55 05

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Is the branch installed on stage already?

**Comment by dnil** [comment]
> Is the branch installed on stage already?

It was a couple of minutes ago at least! 😄

File: scout/server/blueprints/institutes/views.py [hunk 1]
```diff
@@ -137,7 +137,9 @@ def causatives(institute_id):
 
     variants = list(store.check_causatives(institute_obj=institute_obj, limit_genes=hgnc_id))
     if variants:
-        variants = sorted(variants, key=lambda k: k["hgnc_symbols"])
+        variants = sorted(
+            variants, key=lambda k: k.get("hgnc_symbols", [None])[0] or k.get("str_repid")
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/views.py:141]: Code looks fine, and your test as well!

**Review by northwestwitch — APPROVED** [review]
It works!

**Pull request merged**